src/react-components/modals/loading.jsx

```diff
@@ -0,0 +1,32 @@
+/*
+ * This program is part of the OpenLMIS logistics management information system platform software.
+ * Copyright © 2017 VillageReach
+ *
+ * This program is free software: you can redistribute it and/or modify it under the terms
+ * of the GNU Affero General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or (at your option) any later version.
+ *  
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. 
+ * See the GNU Affero General Public License for more details. You should have received a copy of
+ * the GNU Affero General Public License along with this program. If not, see
+ * http://www.gnu.org/licenses.  For additional information contact info@OpenLMIS.org. 
+ */
+
+import React from 'react';
+
+const Loading = ({ display }) => {
+    const showHideClassName = display ? 'react-modal' : 'display-none';
```

Comment:
I think it would be easier to maintain just the `<Loading />` component or remove it from the code in the target, then there should be no need to specify the `display` parameter, and just specifying the `react-modal` style inside the div should be enough.

Then usage would be similar to this
`isSomethingInProgress && <Loading />`

Reply:
hi @mdulko I adressed your comment